services/docdb/src/main/java/software/amazon/awssdk/services/docdb/internal/execution.interceptors

```diff
@@ -0,0 +1,2 @@
+software.amazon.awssdk.services.docdb.internal.CopyDbClusterSnapshotPresignInterceptor
```

Comment:
Are these two imports needed? What is this class for?

Reply:
This was accidental..removed this